examples/documentation_examples/examples.go

```diff
@@ -2712,3 +2712,100 @@ func IndexExamples(t *testing.T, db *mongo.Database) {
 		require.NoError(t, err)
 	}
 }
+
```

Comment:
You're right that this file is getting pretty large. From my understanding, this is mostly referenced by the Docs team, who pull the examples out automatically (hence the standardized `Start Example` / `End Example`). Filed a ticket GODRIVER- to suggest splitting this file up into smaller ones, so that users might be able to look at these examples more easily.

Reply:
I'm not sure the Docs team will want us to do that, since I think that requires work on their end.

Reply:
Ah, I didn't realize these were being read by a docs tool. Consider adding a comment in the file or a README in the `documentation_examples` dir that describes the purpose of these examples. Otherwise, looks good 👍

Reply:
A README would be nice; noted that in GODRIVER-2004.